pydocx/export/base.py

```diff
@@ -535,3 +539,16 @@ def export_field_char(self, field_char):
 
     def export_field_code(self, field_code):
         pass
+
+    def export_textbox(self, textbox):
+        return self.yield_nested(textbox.children, self.export_node)
+
+    def export_textbox_content(self, textbox_content):
+        return self.yield_nested(textbox_content.children, self.export_node)
+
+    # Markup Compatibility exporters
+    def export_alternate_content(self, alternate_content):
```

Comment:
Could we call this `export_markup_compatibility_alternative_content`? I'd rather have a longer-than-ideal name that explains the concept than a name that needs a comment.